deps/wazuh_testing/wazuh_testing/process_resource_monitoring/src/process_resource_monitoring/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""Process resource usage monitoring tool.
+
+This package contains the following modules:
+
+monitor -- Process and child processes monitoring using one process per
+           instance (one thread). Tracks several resource usage metrics
+
+disk_usage_tracker -- Track disk usage of files/directories over time. Show 
+                      absolute and relative (to the partition) usage.
+
+"""
+
+from process_resource_monitoring.disk_usage_tracker import DiskUsageTracker
+from process_resource_monitoring.monitor import Monitor
+
```

Comment:
Not sure if the convention we are following instructs to leave 2 blank lines after the import blocks but with the current configuration I am using for the linter, it is marked as an error.
```
src/process_resource_monitoring/__init__.py:18:1: I001 [*] Import block is un-sorted or un-formatted
   |
18 | / from process_resource_monitoring.disk_usage_tracker import DiskUsageTracker
19 | | from process_resource_monitoring.monitor import Monitor
20 | | 
21 | | 
22 | | __all__ = ['Monitor', 'DiskUsageTracker']
   | |_^ I001
   |
   = help: Organize imports

Found 1 error.
[*] 1 fixable with the `--fix` option.
```

Reply:
We have been using that style in other PRs. Even so, since the styles are not completely defined, I think we can leave it like that and if @Rebits  thinks that the change should be made, he will let you know in his review.